README.md

```diff
@@ -1,8 +1,7 @@
 # Capstone
```

Comment:
- [ ] Add the correct project name in the Readme file.
- [ ] Also, it'll be nice if you could add more information about the authors as well.

Reply:
done 👍